### PR TITLE
mount-tool: trivial tweaklets

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

**Review by yuwata — APPROVED** [review]

File: src/mount/mount-tool.c
Comment (yuwata) [inline src/mount/mount-tool.c]: Oh? What? Why `COLUMN_VENDOR` is inserted?

Comment (poettering) [reply src/mount/mount-tool.c]: yeah, accident. dropped it.

(i added a vendor column based on #30009, but that needs to wait for another time, once that PR is merged)

**Comment by poettering** [comment]
fixed the "vendor" thing, setting green label again, since fix was so trivial

**Review by yuwata — APPROVED** [review]

**Comment by yuwata** [comment]
(semaphoreci is now unstable, and the failure seems to be timeout. Unrelated.)

**Pull request merged**
